### PR TITLE
Fix: Broken link

### PR DESCRIPTION
**Pull request by SatvikG7**

### CODE REVIEW

**Comment by SatvikG7** [comment]
@Sneh1999

File: README.md [hunk 1]
```diff
@@ -51,4 +51,4 @@ In this module you will learn about what are Crypto Wallets and how to download
   - [Trust Wallet](https://trustwallet.com/)
   - [Atomic Wallet](https://atomicwallet.io/)
     
-   P.S. - To connect other wallets in LearnWeb3DAO you have to use [Wallet Connect](https://walletconnect.com/).Here is the list of all [wallets supported by WalletConnect](https://walletconnect.com/registry/wallets).
+   P.S. - To connect other wallets in LearnWeb3DAO you have to use [Wallet Connect](https://walletconnect.com/).Here is the list of all [wallets supported by WalletConnect](https://walletconnect.com/registry?type=wallet).
```
Comment (Sneh1999) [inline README.md:54]: Can you add your discord id?  I would like to give you the contributor role on discord 😄?

Comment (SatvikG7) [reply README.md:54]: Discord: SatvikG7#3180

Comment (Sneh1999) [reply README.md:54]: I cant find you on LearnWeb3's discord. Tag me so that I can give you the role

Comment (SatvikG7) [reply README.md:54]: I have DMed you just now.

**Pull request merged**